src/Test/QuickCheck/Laws/Data/BoundedOrd.purs

```diff
@@ -1,18 +1,23 @@
-module Test.QuickCheck.Laws.Data.Bounded where
-
-import Console (log)
-import Test.QuickCheck (QC(..), quickCheck)
-import Test.QuickCheck.Arbitrary (Arbitrary, Coarbitrary)
-import Type.Proxy (Proxy())
-
--- | - Ordering: `bottom <= a <= top`
-checkBounded :: forall a. (Arbitrary a, Bounded a) => Proxy a -> QC Unit
-checkBounded _ = do
-
-  log "Checking 'Ordering' law for Bounded"
-  quickCheck ordering
-
-  where
-
-  ordering :: a -> Boolean
-  ordering a = bottom <= a && a <= top
+module Test.QuickCheck.Laws.Data.Bounded where
```

Comment:
We should probably make this `BoundedOrd` as the law applies to that specifically.